### PR TITLE
test

### PR DESCRIPTION
**Pull request by Junochop**

### CODE REVIEW

**Comment by morecallan** [comment]
@Junochop - Make sure to get this deployed, too.

File: partials/favorites.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="text-center" id="log">
```
Comment (morecallan) [inline partials/favorites.html]: Make sure this is just doing/showing "Hello Favorites" (basically what you did in contacts/new and contacts/view and login).

**Review by morecallan — COMMENTED** [review]
@Junochop Close! Just make the one suggested change, push up and I'll merge it!

File: partials/favorites.html (hunk 1 above)
Comment (Junochop) [reply partials/favorites.html]: done

**Comment by rousell** [comment]
@Junochop 👍 change looks good! merging in

**Pull request merged**